### ISSUE TITLE
bots stay in spawn if a player is too close

### ISSUE THREAD

**Issue by snipergaming134** [explowz/cathook#34]
gui just has the bot go a different way, ipc just has the bot stay in spawn until the player moves away in distance from spawn

**Comment by snipergaming888** [comment]
this is true, also happens to me, but it happens on all nullworks versions. (i think?)
